JDStatusBarNotification/JDStatusBarNotification.m

```diff
@@ -481,16 +481,19 @@ - (void)updateWindowTransform;
 
 - (void)updateTopBarFrameWithStatusBarFrame:(CGRect)rect;
 {
-  CGFloat width = MAX(rect.size.width, rect.size.height);
-  CGFloat height = MIN(rect.size.width, rect.size.height);
-
-  // on ios7 fix position, if statusBar has double height
-  CGFloat yPos = 0;
-  if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 7.0 && height > 20.0) {
-    yPos = -height/2.0;
-  }
-
-  _topBar.frame = CGRectMake(0, yPos, width, height);
+    CGFloat width = MAX(rect.size.width, rect.size.height);
+    CGFloat height = MIN(rect.size.width, rect.size.height);
+    
+    // on ios7 fix position, if statusBar has double height
+    CGFloat yPos = 0;
+    if ([JDStatusBarView isIphoneX]) {
+        height = 50;
```

Comment:
we should never hardcode any size.. can we read the layoutGuides somehow? How will this work in landscape?